android/src/main/java/com/dormmom/flutter_twilio_voice/IncomingCallNotificationService.java

```diff
@@ -80,20 +80,25 @@ private Notification createNotification(CallInvite callInvite, int notificationI
         Log.i(TAG, "Setting notification from, "+ callInvite.getFrom());
         String fromId = callInvite.getFrom().replace("client:","");
         String caller = preferences.getString(fromId, preferences.getString("defaultCaller", "Unknown caller"));
-
+        String callerName;
+        if(isCallAccepted){
+            callerName = "CallAccepted";
```

Comment:
please add a space in between Call and Accepted

Reply:
ok